### ISSUE TITLE
Random jumps in language downloading menu.

### ISSUE THREAD

**Issue by Podiumnoche** [mkiol/dsnote#29]
`Flatpak` `4.1.0`

After clicking download, scroll is jumping to top of list (most often if while clicked it was scrolled down), then when i will click download to some model at top, scroll may jump down.

Nothing is showing on output when runned with `dsnote --verbose` command, except following, but these occur only when opening languages menu.

```
[W] 20:33:49.225 0x7fe9ee845d80 () -   OpenType support missing for "Unifont", script 12
[W] 20:33:49.312 0x7fe9d1066600 () -   OpenType support missing for "Unifont", script 12
[W] 20:33:49.371 0x7fe9ee845d80 () -   OpenType support missing for "Biwidth", script 11
[W] 20:33:49.380 0x7fe9ee845d80 () -   OpenType support missing for "Fixed", script 11
[W] 20:33:49.398 0x7fe9d1066600 () -   OpenType support missing for "Biwidth", script 11
[W] 20:33:49.407 0x7fe9d1066600 () -   OpenType support missing for "Fixed", script 11
```

**Comment by mkiol** [comment]
Thanks for pointing this.

Fix: 6d89fde931e18fdb5c020aaf01e3521e5a1ac516

Will be included in upcoming release.

**Comment by mkiol** [comment]
Fixed in version 4.2.0.

6d89fde931e18fdb5c020aaf01e3521e5a1ac516
